man/man8/zfs.8

```diff
@@ -718,6 +718,13 @@ This property can be either
 .Sy yes
 or
 .Sy no .
+.It Sy objsetid
+A unique identifier for this dataset within the pool. Unlike the dataset's
+.Sy guid
+, the
+.Sy objsetid
+of a dataset is not transferred to other pools when the snapshot is copied
+with a send/receive operation.
```

Comment:
This should be rarely used, outside of the case for kstats. Do you think it is worth mentioning here that the objsetid can be referenced to the kstats?

Reply:
You might also mention that the objsetid can be reused (for a new dataset) after the dataset is deleted.